README.md

```diff
@@ -15,7 +15,10 @@ gem 'counter_cache-rails'
 ```rb
 class Post
   has_many :comments
-  counter_cache :comments
+
+  counter_cache :comments,
+                if: ->(comment) { comment.visible? },
```

Comment:
I think rails programmer expects `:unless` option is available when `:if` option was available, but, I think this option's mainly use case is that judgment target record is countable whether or not.

So, I guess the option should be named to "countable". what do you think?